CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# dbt_netsuite v0.13.0
+[PR #116](https://github.com/fivetran/dbt_netsuite/pull/116) includes the following updates: 
+
+## 🚨 Breaking Changes 🚨
```

Comment:
i feel like we should also call out the upstream breaking changes around the staging model materializations

Reply:
Thanks added!